### DISCUSSION TITLE
Render Texture

### DISCUSSION THREAD

**Discussion by sabanadesan** [StereoKit/StereoKit#631] [Q&A]
How do you render an image onto the surface of one side of an object like a cube?

**Comment by maluoi** [comment]
I'd recommend checking out the camera widget:

https://github.com/StereoKit/StereoKit/blob/master/Examples/StereoKitTest/Tools/RenderCamera.cs

Or the mirror demo!

https://github.com/StereoKit/StereoKitReleaseNotes/blob/main/v0.3.1/SceneRenderToTex.cs
